### PR TITLE
give more details in the discussions opened by parquet-converter

### LINKED ISSUE #2419: Improve the message in Parquet conversion discussions
see https://huggingface.slack.com/archives/C02V51Q3800/p1707388935912969 (internal)

> Would it make sense to add some message in Parquet PRs (e.g. https://huggingface.co/datasets/huggan/smithsonian_butterflies_subset/discussions/2#65c41c1580497543ca3f8a5a) for less technical people?
> As an aside, I don't think the docs really mention the advantages of Parquet even if we link there for advantages (https://huggingface.co/docs/datasets-server/parquet)
> For example
> ```
> [bot] [No action needed] Conversion to Parquet
> 
> The parquet-converter bot has created a version of this dataset in the Parquet format in the `refs/convert/parquet` branch. This will allow X, Y, Z.
> 
> You can learn more about the advantages associated with this format in the documentation.
> ```

related to https://github.com/huggingface/datasets-server/issues/2349#issuecomment-1916292176

### PR DESCRIPTION
**Pull request by severo**
fixes #2419

It could be too verbose now.

Also: we might add a section or a sentence to handle the case where the dataset was already in Parquet (until we properly detect if the dataset was already in Parquet format: https://github.com/huggingface/datasets-server/issues/2301)

### CODE REVIEW

**Review by stevhliu — COMMENTED** [review]
I wonder if maybe it'd be better to add a link to the content you're adding in #2422 so users can follow it to learn more details if they're interested and so that it isn't as verbose?

**Comment by severo** [comment]
Indeed. I already link to that doc page. But also... I have the feeling people don't often click to read details, so... I wanted to add part of the text here. I'll update with the reviews on that other PR, and also try to reduce a bit, maybe.

**Review by davanstrien — COMMENTED** [review]
Maybe it's worth linking to the full doc for the more detailed explanation of parquet in the discussion focus on a few key benefits (maybe just 3 main bullet points) Something like:

The main benefits of Parquet are:
- Reduced memory requirements
- Compatiability with datasets-server 
- etc.

**Comment by severo** [comment]
good feedback, thanks. It tried to improve it

**Review by osanseviero — APPROVED** [review]
Nice!

File: jobs/cache_maintenance/src/cache_maintenance/discussions.py [hunk 1]
```diff
@@ -13,6 +13,35 @@
...
 DAYS = 1
 
+DISCUSSION_TITLE = "[bot] [No action needed] Conversion to Parquet"
+DISCUSSION_DESCRIPTION = """The {bot_name} bot has created a version of this dataset in the Parquet format in the {parquet_link} branch.
+
+## What is Parquet?
+
+Apache Parquet is a popular columnar storage format known for:
+
+- reduced memory requirement,
```
Comment (stevhliu) [inline jobs/cache_maintenance/src/cache_maintenance/discussions.py:23]: Super nice, I think this abbreviated version is a lot better! 🔥

**Review by stevhliu — APPROVED** [review]

**Comment by severo** [comment]
it's live: 

https://huggingface.co/datasets/ErfanMoosaviMonazzah/esnli-seq2seq/discussions/1#65c736010831983147feaf47

Capture d’écran 2024-02-12 à 10 56 05

**Comment by julien-c** [comment]
updated comment looks good, here are a few small tweaks:

```md
**This is what powers the Dataset Viewer** on each dataset page and every dataset on the Hub can be accessed with the same code (you can use HF Datasets, ClickHouse, DuckDB, Pandas or Polars, [up to you](https://huggingface.co/docs/datasets-server/parquet_process)). 

[...]

```

> This rule suffers an exception

This rule has an exception

**Comment by severo** [comment]
done: https://github.com/huggingface/datasets-server/pull/2441

**Pull request merged**